src/System.Security.Cryptography.Xml/src/project.json

```diff
@@ -7,7 +7,7 @@
     },
     "net463": {
```

Comment:
@steveharter Any project which existed in .NET Core 1.1 already built facades, and now they're getting pulled through in the packaging already, so they only are targeting future builds for new API.  Since this is a new package, it needs to set the baseline for facading.  In this case, 4.6.2